src/core/math/quat.js

```diff
@@ -301,6 +315,35 @@ class Quat {
         return this.x * this.x + this.y * this.y + this.z * this.z + this.w * this.w;
     }
 
+
+    /**
+     * Performs a linear interpolation between two quaternions. The result of the interpolation
+     * is written to the quaternion calling the function.
```

Comment:
Worth adding a comment here (and in slerp) about when you would use one over the other?